explore_simulation_approach/4_plot.py

```diff
@@ -21,8 +21,47 @@
 
 # %load_ext autoreload
 # %autoreload 2
+# %matplotlib inline
+import os
 import pandas as pd
 import seaborn as sns
+from ponyo import utils
+import pickle
+
+# ## Visualize the simulated data
+
+# +
+# Read in config variables
+base_dir = os.path.abspath(os.path.join(os.getcwd(), "../"))
+
+config_filename = "config_sophie_vs_trad.tsv"
+
+params = utils.read_config(config_filename)
+
+# +
+# Load config params
+
+# Local directory to store intermediate files
+local_dir = params["local_dir"]
+
+#
+dataset_name = params["dataset_name"]
+
+# File containing un-normalized template experiment
+raw_template_filename = params["raw_template_filename"]
+
+# ID for template experiment to be selected
+project_id = params["project_id"]
+
+# Identifier for simulated experiment
+i = 0
+
+# Pickle files saving specific and generic gene ids
+template_specific_gene_ids_filename = params["template_specific_gene_ids_filename"]
+generic_gene_ids_filename = "generic_gene_ids.pickle"
+# -
+
+# ## STOP
```

Comment:
Do you want to elaborate a bit here? Maybe just say you decided to use the plots in `3_evaluate.ipynb` and these are out of date?

Reply:
This notebook can be removed now.